ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -89,22 +92,30 @@ def get_segment_order(unaligned_sequences: dict[str, str]) -> list[str]:
     return sorted(segment_order)
 
 
-def get_address(config: Config, entry: dict[str, str]) -> str:
+def get_address(config: Config, entry: dict[str, Any]) -> str:
     address_string = entry["center_name"]
     if config.is_broker:
         try:
-            group_info = get_group_info(config, entry["metadata"]["groupId"])[0]["group"]
-            address = group_info["address"]
-            address_list = [
-                entry["center_name"],  # corresponds to Loculus' "Institution" group field
-                address.get("city"),
-                address.get("state"),
-                address.get("country"),
+            group_info = call_loculus.get_group_info(config, entry["metadata"]["groupId"])[0][
+                "group"
```

Comment:
lets also add the `address = group_info["address"]` inside the try catch (as this assumes the response format of `get_group_info` includes that field)